ts/screens/profile/ProfileMainScreen.tsx

```diff
@@ -230,6 +247,32 @@ const ProfileMainScreenFC = () => {
       />
       <VSpacer size={8} />
       <ContentWrapper>
+        {appFeedbackEnabled && surveyUrl && (
+          <ListItemAction
+            label={feedbackLabel}
+            icon="message"
+            variant="primary"
+            testID="feedbackButton"
+            onPress={() => {
+              void openAuthenticationSession(surveyUrl, "");
+            }}
+            accessibilityLabel={feedbackLabel}
+          />
+        )}
+        <ListItemAction
+          label={reviewLabel}
+          icon="starEmpty"
+          variant="primary"
+          testID="reviewButton"
+          onPress={Platform.select({
+            ios: () =>
+              openWebUrl(
+                "https://apps.apple.com/app/id1501681835?action=write-review"
+              ),
```

Comment:
On iOS, if we fail to open the URL for some reason, we don’t catch the exception. Is it okay not to alert the user in this case?